controllers/admin/AdminPreferencesController.php

```diff
@@ -188,15 +188,22 @@ public function __construct()
                     'class' => 'fixed-width-xxl',
                 ),
                 'PS_DISPLAY_SUPPLIERS' => array(
-                    'title' => $this->trans('Display brands and suppliers', array(), 'Admin.Shopparameters.Feature'),
-                    'desc' => $this->trans('Enable brands and suppliers pages on your front office even when their respective modules are disabled.', array(), 'Admin.Shopparameters.Help'),
+                    'title' => $this->trans('Display suppliers', array(), 'Admin.Shopparameters.Feature'),
```

Comment:
Do you really need to change this legacy controller? I thought this page was already migrated to `/configure/shop/preferences/preferences`

Reply:
I added it here fearing that not doing so might break something. If this controller is fully deprecated then yes, there's not need to modify it.

Reply:
Now that it's done I think you can leave, but were you able to test it? Because if the routing is properly set with `_legacy_link` you shouldn't be able to access the legacy page any more

Reply:
You're right, I couldn't test a page that is not linked. I just knew that the 'Display brands' translation had to be reworded, and I was not sure about the consequences if I left the old wording untouched at this place.